src/sdk/namespace_fs.js

```diff
@@ -667,7 +680,7 @@ class NamespaceFS {
         }
     }
 
-    async _upload_stream(source_stream, upload_path, fs_account_config, fs_xattr) {
+    async _upload_stream(source_stream, upload_path, fs_account_config, rpc_client, fs_xattr) {
```

Comment:
Why not use the rpc_client in this (this.rpc_client) like we do in other namespaces in the constructor?

Reply:
We don't have rpc_client here except from the operator_sdk.

Reply:
Pass if to the constructor of the NamespaceFS as we do in NamespaceBlob and S3 and others.
Is there a reason not to?

Reply:
I don't see any reason to pass it.
Actually, on the NamespaceBlob and S3 and others, there is also no reason I see and refactor it to remove it and all those calls.
Do you see a good reason to keep rpc_client in the contractor while we have it on the operator_sdk?